scaler/handlers_test.go

```diff
@@ -88,9 +93,140 @@ func TestIsActive(t *testing.T) {
 			lggr := logr.Discard()
 			table := routing.NewTable()
 			ticker, pinger, err := newFakeQueuePinger(ctx, lggr)
+			r.NoError(err)
+			defer ticker.Stop()
+			tc.setup(table, pinger)
+
+			hdl := newImpl(
+				lggr,
+				pinger,
+				table,
+				123,
+				200,
+			)
+
+			bufSize := 1024 * 1024
+			lis := bufconn.Listen(bufSize)
+			grpcServer := grpc.NewServer()
```

Comment:
```suggestion
			grpcServer := grpc.NewServer()
			defer grpcServer.Stop()
```

if you do this, then likely need to remove the `log.Fatalf` below if the `grpcServer.Serve(lis)` function returns a non-nil error